### DISCUSSION TITLE
Re-mapping of channels

### DISCUSSION THREAD

**Discussion by trexcarey** [ExpressLRS/ExpressLRS#2058] [Ideas]
Hi, I would like to use ELRS for a large-scale helicopter; however, I need a minimum of five full-res channels and the ability to remap the channels, as most FlyBarLess units use channel 5 for gain adjustment. I have been experimenting with Matek Sbus converters and a 10-channel PWM output board. However, I need the ability to remap channels in the Sbus output so that the FBL controller sees the correct information.
I have seen several threads about this issue on chat sites. Is there any way this can be achieved within the programming?
Can I just take this opportunity to thank all the Devs for their hard work and quick response to requests within the community. Hopefully Andy

**Comment by robhaswell** [comment]
I'm using a custom branch build to achieve this: https://github.com/ExpressLRS/ExpressLRS/compare/master...robhaswell:ExpressLRS:swap-channel-5-9#diff-b9d729e1b9f3ef23170ff4793eb43a1aa25f39598b2a78eea778099b6e776393
